runtime/src/append_vec.rs

```diff
@@ -379,36 +378,21 @@ impl<'a> serde::de::Visitor<'a> for AppendVecVisitor {
         let file_size: u64 = deserialize_from(&mut rd).map_err(Error::custom)?;
         let offset: usize = deserialize_from(&mut rd).map_err(Error::custom)?;
 
-        let split_path: Vec<&str> = path.to_str().unwrap().rsplit('/').collect();
-        let account_paths = ACCOUNT_PATHS.lock().unwrap().clone();
-        let mut account_path = path.clone();
-        if split_path.len() >= 2 {
-            for dir_path in account_paths.iter() {
-                let fullpath = format!("{}/{}/{}", dir_path, split_path[1], split_path[0]);
-                let file_path = Path::new(&fullpath);
-                account_path = file_path.to_path_buf();
-                if file_path.exists() {
-                    break;
-                }
-            }
-        }
-
         let data = OpenOptions::new()
             .read(true)
             .write(true)
             .create(false)
-            .open(account_path.as_path());
+            .open(&path);
 
         if data.is_err() {
-            warn!("account open {:?} failed", account_path);
-            std::fs::create_dir_all(&account_path.parent().unwrap())
-                .expect("Create directory failed");
-            return Ok(AppendVec::new(&account_path, true, file_size as usize));
+            warn!("account open {:?} failed", &path);
```

Comment:
based on DM conversation, I think this should just error out so we catch it immediately as opposed to having it show up as a downstream ANF or something